source/vir/src/ast.rs

```diff
@@ -256,12 +256,20 @@ pub enum UnaryOp {
     CharToInt,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, Hash, PartialOrd, Ord, ToDebugSNode)]
+pub enum VariantCheck {
+    None,
+    //Recommends,
```

Comment:
This is great in preparation for https://github.com/verus-lang/verus/pull/889. Thanks!